rotkehlchen/chain/ethereum/manager.py

```diff
@@ -555,15 +590,25 @@ def _ens_lookup(self, web3: Optional[Web3], name: str) -> Optional[ChecksumEthAd
         )
         if is_none_or_zero_address(resolver_addr):
             return None
+
+        ens_resolver_abi = list(ENS_RESOLVER_ABI)
+        arguments = [normal_name_to_hash(normal_name)]
+        if blockchain != SupportedBlockchain.ETHEREUM:
+            ens_resolver_abi.extend(ENS_RESOLVER_ABI_MULTICHAIN_ADDRESS)
+            arguments.append(blockchain.ens_coin_type())
+
         address = self._call_contract_etherscan(
             to_checksum_address(resolver_addr),
-            abi=ENS_RESOLVER_ABI,
+            abi=ens_resolver_abi,
             method_name='addr',
-            arguments=[normal_name_to_hash(normal_name)],
+            arguments=arguments,
         )
-
         if is_none_or_zero_address(address):
             return None
+
+        if blockchain != SupportedBlockchain.ETHEREUM:
+            return HexBytes(address)
```

Comment:
I didn't want to use `HexStr` because it comes from the `eth_typing` library, and it seemed wrong to me to start importing it in non-Ethereum related files/modules.
The HexBytes documentation allegedly says that it is a very light wrapper... I can't tell. The thing is that `addr(bytes32,uint256` returns `bytes`. I either could create our own alias of `str` or just return a `str`.

Reply:
Hmm I don't understand why not just return a string here? In the end you take HexBytes.hex() so returning HexBytes is not needed.